uhabits-core/src/jvmMain/java/org/isoron/uhabits/core/models/ScoreList.kt

```diff
@@ -68,19 +68,19 @@ class ScoreList {
     fun recompute(
         frequency: Frequency,
         isNumerical: Boolean,
+        numericalHabitType: NumericalHabitType,
         targetValue: Double,
         computedEntries: EntryList,
         from: Timestamp,
         to: Timestamp,
     ) {
```

Comment:
Could you please write some unit tests to check that scores for AT_MOST habits are being computed correctly?

Reply:
I don't think there are a lot of tests for numerical as is.. How complex do you want them?